src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -273,7 +273,7 @@ doLedgerEntry(Context const& context)
         if (!state.contains(JS(currency)) ||
             !state.at(JS(currency)).is_string())
         {
-            return Status{Error::rpcINVALID_PARAMS, "malformedCurrency"};
+            return Status{Error::rpcMALFORMED_CURRENCY, "malformedCurrency"};
```

Comment:
You added the error codes header from `build` but forgot to add `rpcMALFORMED_CURRENCY`? Or perhaps it got overwritten by a rebuild of `clio`. Either way this does not work.

Reply:
@godexsoft let's talk further on how to rethink Clio custom error codes as this is the only viable solution forward